src/esp/assets/ResourceManager.cpp

```diff
@@ -2962,6 +2962,16 @@ void ResourceManager::loadTextures(Importer& importer,
           format = Mn::GL::textureFormat(image->compressedFormat());
         } else {
           format = Mn::GL::textureFormat(image->format());
+          // Modify swizzle for single channel textures so that they are
+          // greyscale
+          if (pixelFormatSize(image->format()) == 1) {
+#ifdef MAGNUM_TARGET_WEBGL
+            ESP_WARNING() << "Single Channel Texture ID" << iTexture
+                          << "Loaded as RGB";
```

Comment:
This isn't really saying what it will result in. It should mention something about the texture being wrongly red because greyscale expansion isn't implemented here.

As I said on Slack, on WebGL this is doable (you'd do the channel expansion CPU side) but needs a tiny helper to  invent the corresponding expanded pixel format enum value. I'm in the process of adding that.